### PR TITLE
🪟 🐛 🎨 Fix design issues with CatalogDiff modal

### PR DESCRIPTION
**Pull request by edmundito**
## What
Fixes the catalog diff modal size along with some styling issues with how fields were being displayed.

Before:
Screen Shot 2022-11-02 at 15 38 10

After:
Screen Shot 2022-11-04 at 15 53 17

## How
* Improved paddings and gaps
* Field table was manually controlling the margins around the accordion, now the accordion pushes the table inward
* Fix CatalogDiff storybook
* Update text colors of Number badge to be consistent
* Remove relative paths from scss files

## Recommended reading order
Top to bottom

### CODE REVIEW

File: airbyte-webapp/src/views/Connection/CatalogDiffModal/components/DiffHeader.tsx [hunk 1]
```diff
@@ -11,15 +11,17 @@ interface DiffHeaderProps {
...
+  const { formatMessage } = useIntl();
+
+  const text = formatMessage(
+    {
+      id: `connection.updateSchema.${diffVerb}`,
+    },
+    {
+      value: diffCount,
+      item: formatMessage({ id: `connection.updateSchema.${diffType}` }, { count: diffCount }),
+    }
```
Comment (edmundito) [inline airbyte-webapp/src/views/Connection/CatalogDiffModal/components/DiffHeader.tsx:23]: Nit change: This was rendered strangely in the inspector. The string was split into 3 parts in the dom. This change will render the string as a whole.

**Review by krishnaglick — COMMENTED** [review]
Can you update this against master? This doesn't have a start:cloud right now and I've deleted my .env.development so I can't test this fully.

File: airbyte-webapp/src/components/ui/NumberBadge/NumberBadge.module.scss [hunk 2]
```diff
@@ -17,23 +17,21 @@
   font-style: normal;
   font-weight: 500;
   font-size: 10px;
+  color: colors.$white;
```
Comment (timroes) [inline airbyte-webapp/src/components/ui/NumberBadge/NumberBadge.module.scss:20]: This is the wrong text color for this background color. It's not fulfilling WCAG AA contrast ratios. The supposed text color on those background colors were correct before this PR.

**Review by timroes — DISMISSED** [review]

File: airbyte-webapp/src/components/ui/NumberBadge/NumberBadge.module.scss (hunk 2 above)
Comment (edmundito) [reply airbyte-webapp/src/components/ui/NumberBadge/NumberBadge.module.scss]: This was not an arbitrary change, but the current design has not been updated to reflect this. Neither @Upmitt nor I were aware that this change was made for this reason. I have reverted the change.

File: airbyte-webapp/src/views/Connection/CatalogDiffModal/index.stories.tsx [hunk 3]
```diff
@@ -12,15 +14,17 @@ export default {
...
-      />
-    </Modal>
+    <ModalServiceProvider>
+      <Modal size="md" title={<FormattedMessage id="connection.updateSchema.completed" />}>
+        <CatalogDiffModal
+          catalogDiff={args.catalogDiff}
+          catalog={args.catalog}
+          onClose={() => {
+            return null;
+          }}
```
[the remark this answers is not in the document]
Comment (edmundito) [reply airbyte-webapp/src/views/Connection/CatalogDiffModal/index.stories.tsx:24]: Not optional for catalog diff. Will clean it up.

**Comment by krishnaglick** [comment]
Screenshot 2022-11-14 at 2 46 51 PM
Could maybe use a little extra padding on the bottom, but otherwise LGTM!

**Review by krishnaglick — APPROVED** [review]

**Comment by edmundito** [comment]
@krishnaglick Added the padding and looks better, and I updated the storybook to preview with different combinations

**Review by krishnaglick — APPROVED** [review]
LGTM!

**Pull request merged**